pandas/core/generic.py

```diff
@@ -4522,6 +4530,102 @@ def copy(self, deep=True):
         Returns
         -------
         copy : type of caller
+
+        Examples
+        --------
+        >>> s = pd.Series([1,2], dtype="int32", index=["a", "b"])
+        >>> s
+        a    1
+        b    2
+        dtype: int32
+        >>> s_copy = s.copy()
+        >>> s_copy
+        a    1
+        b    2
+        dtype: int32
+
+        Shallow copy versus default (deep) copy:
```

Comment:
Since this is a "section header" can you get it to render in bold (i.e. between asterisks)?